pkg/kubelet/pluginmanager/operationexecutor/nestedpendingoperations.go

```diff
@@ -0,0 +1,252 @@
+/*
+Copyright 2019 The Kubernetes Authors.
```

Comment:
Instead of creating yet another `nestedpendingoperations` either 1) modify the plugin manager to use `goroutinemap` and modify `goroutinemap` to handle `volumetypes.GeneratedOperations` or 2) use the existing `nestedpendingoperations` instead of `goroutinemap` and just set `volumeName` to `socketPath` and leave `podName` empty.

Ideally we should also 1) refactor `nestedpendingoperations` and `goroutinemap` in to a single generic/resuable class, and 2) move `volumetypes.GeneratedOperations` into `nestedpendingoperations` or `goroutinemap` or the proposed combined (refactored) class.